src/injector.js

```diff
@@ -0,0 +1,132 @@
+const puma = require('pumascript');
+const http = require('http');
+const fs = require('fs');
+
+let errorsLogs = [];
+
+/**
+ * Read a .json file with the dependencies to test pumascript.
+ * @param {string} dir - address where the .json file is located.
+ * @return {Object} - object with the content of the json file.
+ */
+function readDependencyJsonFile(dir) {
+    let fileContent;
+    try {
+        fileContent = JSON.parse(fs.readFileSync(dir, 'utf8'));
+    } catch (error) {
+        console.warn(`The file entered does not exist. ${JSON.stringify(error)}`);
+    }
+    return fileContent;
+}
+
+/**
+ * It allows to read the body of a url.
+ * @param {object} options - Object that has to contain host and path 
+ * of the CDN to which you want to access.
+ * @param {fucntion(body)} callback - Function that receives as a parameter the body of the CDN.
```

Comment:
Small typo: `function` :)

Also callbacks are treated different on jsdoc. They are documented separately: http://usejsdoc.org/tags-callback.html
Just so you know, i would leave it like it is. Is easier to read.

Reply:
haha you have reason, I already corrected it.